spring-context/src/main/java/org/springframework/cache/interceptor/CacheOperationExpressionEvaluator.java

```diff
@@ -67,6 +69,8 @@
 
 	private final Map<ExpressionKey, Expression> unlessCache = new ConcurrentHashMap<>(64);
 
+	private final Map<ExpressionKey, Expression> nameCache = new ConcurrentHashMap<>(64);
```

Comment:
Is there any reason why we can't just declare a single `Map<ExpressionKey, Expression>` inside the parent `CachedExpressionEvaluator`?